recipes-devtools/gie/tensorrt-samples_7.1.3-1.bb

```diff
@@ -0,0 +1,72 @@
+DESCRIPTION = "NVIDIA TensorRT Samples for deep learning"
+LICENSE = "Proprietary"
+
+inherit l4t_deb_pkgfeed cuda
+
+HOMEPAGE = "http://developer.nvidia.com/tensorrt"
+
+PREFIX = "NoDLA-"
+PREFIX_tegra194 = "DLA-"
+
+L4T_DEB_GROUP = "tensorrt"
+
+SRC_SOC_DEBS = "\
+    libnvinfer-samples_${PV}+cuda10.2_all.deb;downloadfilename=${PREFIX}libnvinfer-samples_${PV}+cuda10.2_all.deb;name=samples;subdir=tensorrt \
+"
+
+SRC_URI_append = " file://0001-Makefile-fix-cross-compilation-issues.patch"
+
+SRC_URI[samples.sha256sum] = "3a5162474cce7e191d78703b196bc8bdd5a6908d7537200194f1e80d72cf45a0"
```

Comment:
My Xavier build failed because the t194 (DLA) version of the `.deb` has a different SHA256 checksum... although the content appears to be identical to the t186/t210 (non-DLA) version.

Still, we should follow the pattern of the other tensorrt recipes in this branch.  Something like:
```
SAMPSHA256SUM = "3a5162474cce7e191d78703b196bc8bdd5a6908d7537200194f1e80d72cf45a0"
SAMSHA256SUM_tegra194 = "87d32bb231c19717c4690e05c770eb51d3d5ad4bea8fbd0ee46ba4d1a386ad9f"
SRC_URI[samples.sha256sum] = "${SAMPSHA256SUM}"
```
so we use the right checksum based on which copy of the `.deb` is being used.

This isn't an issue with L4T R32.6.x/JetPack 4.4, because NVIDIA thankfully fixed the tensorrt packages so there's just one set that covers both the DLA and non-DLA platforms.

Reply:
Added the checksum for DLA